app/services/processors/specifications.rb

```diff
@@ -258,7 +258,11 @@ def self.create_terms(specification, spec)
     ###
     def self.create_one_term(node)
       # Retrieve the term, if not found, create one with these properties
-      term = Term.find_or_initialize_by(uri: node["@id"], name: Parsers::Specifications.read!(node, "label"))
+      term = Term.find_or_initialize_by(
+        uri: node["@id"],
```

Comment:
Are terms' `uri`s unique (at least within the organization's scope)?

Reply:
@excelsior term's `uri`s are unique, indeed.